### PR TITLE
Auction VC Design Review pt1

### LINKED ISSUE #1208: auction preview state
current
image

update
image

### PR DESCRIPTION
**Pull request by orta**
Got a meeting on deciding some of the events today, so waiting on that to add the events

Closes #1208 
Does a partial subset of #1209

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">3 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td><p>PR is classed as Work in Progress</p>
</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><p>Needs testing on a Phone if change is non-trivial</p>
</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARArtworkFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkFavoritesNetworkModelSpec</a> in <code>test_getFavorites__makes_request_if_no_request_is_in_progress</code></li>
<li><a href="https://github.com/search?q=ARArtworkViewControllerBuyButton+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkViewControllerBuyButtonSpec</a> in <code>test_buy_button__posts_order_if_artwork_has_no_edition_sets</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__passes_artwork_to_share_controller</code></li>
<li><a href="https://github.com/search?q=ARFairViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARTopMenuViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARTopMenuViewControllerSpec</a> in <code>test_navigation__favorites__logged_in__tab_behavior__already_selected__pops_to_root</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

File: Artsy/App/ARUserActivity.m [hunk 1]
```diff
@@ -34,13 +37,16 @@ + (instancetype)activityForEntity:(id<ARSpotlightMetadataProvider>)entity;
...
+        type = ARUserActivityTypeSale;
     }
+
     NSParameterAssert(type);
 
     ARUserActivity *activity = [[ARUserActivity alloc] initWithActivityType:type];
     activity.title = entity.name;
     activity.webpageURL = [ARSpotlight webpageURLForEntity:entity];
-    activity.userInfo = @{ @"id": entity.publicArtsyID };
+    activity.userInfo = @{@"id" : entity.publicArtsyID};
```
Comment (alloy) [inline Artsy/App/ARUserActivity.m]: boooo

File: Artsy/App/ARUserActivity.m [hunk 2]
```diff
@@ -51,7 +57,7 @@ + (instancetype)activityForEntity:(id<ARSpotlightMetadataProvider>)entity;
                                                             includeIdentifier:NO
                                                                    completion:^(CSSearchableItemAttributeSet *attributeSet) {
             [activity updateContentAttributeSet:attributeSet];
-        }];
+                                                                   }];
```
Comment (alloy) [inline Artsy/App/ARUserActivity.m]: way more boooo

File: Artsy/App/ARUserActivity.m (hunk 1 above)
Comment (orta) [inline Artsy/App/ARUserActivity.m]: Giphy

File: Artsy/App/ARUserActivity.m (hunk 2 above)
Comment (orta) [inline Artsy/App/ARUserActivity.m]: #wontfix

**Comment by alloy** [comment]
Is there any actual analytics changes in this PR? The one commit that seems relevant is https://github.com/artsy/eigen/commit/42f33a4d07a37fe3cba100f4ee0d49a0e3fda93c, but I don’t see how it makes any real changes. And is there a list available of the polish that was required?

**Comment by orta** [comment]
Hrm, well. It started off about analytics + the spotlight integration. Now seems to be mostly about addressing chunks of the design review feedback.

**Comment by alloy** [comment]
:+1: :8ball: :-1:

**Pull request merged**